extensions/stac-eo-spec.md

```diff
@@ -82,6 +82,64 @@ numbers of several popular instruments.
 | lwir11      | 10.5 - 11.5     |           |           | 10        |            | 31    |
 | lwir12      | 11.5 - 12.5     |           |           | 11        |            | 32    |
 
+### Processing level fields
+
+We define **direct conversion** as a process that converts the input data to a physical measurement
+using only information available in the input data and its metadata. The process may be
+an identity function when data is already provided as the physical measurement.
+
+An Item is said to provide
+a processing level feature if there exists a direct conversion from data as presented to the
+physical measurement associated with the processing level.
+
+#### Radiometric processing features
+
```

Comment:
Maybe its better to define a common prefix to group the radiometric features, the same way I did for geometric features below.